libs/backend/src/routes/submission/index.ts

```diff
@@ -1,7 +1,8 @@
 import { FastifyInstance } from "fastify";
 import {
-	AuthenticatedInfo,
 	isAuthenticatedInfo,
+	isPistonExecuteResponseError,
+	isPistonExecuteResponseSuccess,
```

Comment:
```suggestion
	isPistonExecuteResponseError,
	isPistonExecuteResponseSuccess,
```

feel like it should be renamed to `execution` instead of `execute`, minor change, but could be a nice little improvement